protocol.md

```diff
@@ -298,31 +294,24 @@ The Client MAY supply the `Upload-Metadata` header to add additional metadata to
 upload creation request. The Server MAY decide to ignore or use this information
 to further process the request or to reject it.
 
-If the size of the upload exceeds the maximum which MAY be indicated using the
-`Tus-Max-Size` headers, the Server MUST respond with the
-`413 Request Entity Too Large` status code. 
-
-If an upload contains additional metadata responses to `HEAD` requests against
-these uploads MUST include the `Upload-Metadata` header and its value as sent in the
-upload creation request.
```

Comment:
Added this

The Client MAY supply the `Upload-Metadata` header to add additional metadata to the
upload creation request. The Server MAY decide to ignore or use this information to 
further process the request or to reject it. If the Server supports client supplied
metadata it MUST include `Upload-Metadata` in all `HEAD` responses and MUST set the
value to the one specified by the Client during creation.